README.md

```diff
@@ -58,14 +58,19 @@ Then you need to enable this component with the id of the ble_client component.
 
 ```yaml
 uni_trend_sound_meter:
-    # [Required] Reference to the ble_client.id set above 
+    # [Required] Name or ID is required.
+    name: "Sound Level"
+    # [Optional] Enhance entity with icon and unit.
+    icon: 'mdi:microphone-variant'
```

Comment:
How about just use `mdi:microphone` or `mdi:microphone-outline`, as the sound meter is wireless?
The settings below all have default values in the [`__init__.py`](https://github.com/daweizhangau/esphome_uni_trend_sound_meter/blob/main/components/uni_trend_sound_meter/__init__.py#L20-L23). Need to change them as well.